pyglmnet/utils.py

```diff
@@ -130,3 +138,41 @@ def set_log_level(verbose):
             raise ValueError('verbose must be of a valid type')
         verbose = logging_types[verbose]
     logger.setLevel(verbose)
+
+
+def _verbose_iterable(data):
```

Comment:
humm ... does this respect the `verbose` argument that is passed to the `GLM` class?

Reply:
Yes, it does. It basically checks the logger's level and if it is appropriate, then it makes the progress interactive.

Reply:
okay looks good to me then! ready for merge?

Reply:
Yup, I've just changed the PR's title.